packages/stylelint-config/rules/extras.js

```diff
@@ -0,0 +1,87 @@
+    'selector-max-id': 0,
+
+    // No default value
+    // Reason: Declaring font weights in other ways is unsafe.
+    // Docs: https://stylelint.io/user-guide/rules/list/font-weight-notation
+    'font-weight-notation': 'numeric',
+
+    // No default value
+    // Reason: Color-named values disabled - desired lint should check for design system token values only.
+    // Docs: https://stylelint.io/user-guide/rules/list/color-named
+    'color-named': 'never',
+
+    // Default value: 'always' with secondary options.
+    // Reason: Turned off to support grouping variables using empty lines.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-empty-line-before/README.md
+    'scss/dollar-variable-empty-line-before': null,
+
+    // Default value: 'never'
+    // Reasons:
+    // - The aim is unified markdown;
+    // - mixin is a callable function;
+    // - will not be misunderstood with extended selector.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-mixin-argumentless-call-parentheses/README.md
+    'scss/at-mixin-argumentless-call-parentheses': 'always',
+
+    // Default value: true
+    // Reason: Since there is `max-line-length` rule se to max. 120 chars, it should be allowed to use a new line for better code readability.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-newline-after/README.md
+    'scss/operator-no-newline-after': null,
+
+    // Default value: true
+    // Reason: Since there is `max-line-length` rule se to max. 120 chars, it should be allowed to use a new line for better code readability.
+    // (same reason for rule above)
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/operator-no-newline-before/README.md
+    'scss/operator-no-newline-before': null,
+
+    // Default value: 'always'
+    // Reason: Support multi-line value definitions lets developer make code more readable.
+    // It is handy especially for uneasy math calculations.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-colon-space-after/README.md
+    'scss/dollar-variable-colon-space-after': 'always-single-line',
+
+    // Default value `comment-no-empty`: null
+    // Default value `scss/comment-no-empty`: true
+    // Reason: Allow empty lines in structured block comments.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/comment-no-empty/README.md
+    'scss/comment-no-empty': null,
+
+    // Default value: true
+    // Reasons:
+    // - In some cases it is handy to use functions like `unquote`;
+    // - values come from Supernova, so you never know what you're going to get.
+    // Docs: https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/function-unquote-no-unquoted-strings-inside/README.md
+    'scss/function-unquote-no-unquoted-strings-inside': null,
+  },
```

Comment:
Cool! The `scss/*` rules look great. Thanks for putting down all the reasons 👍🏻.